rebus/assetlist.json

```diff
@@ -0,0 +1,27 @@
+{
+	"$schema": "../assetlist.schema.json",
+	"chain_name": "rebus",
+	"assets": [
+      {
+      "description": "REBUS, the native coin of the Rebus chain.",
+      "denom_units": [
+        {
+          "denom": "arebus",
+          "exponent": 0
+        },
+        {
+          "denom": "rebus",
+          "exponent": 18
+        }
+      ],
+      "base": "arebus",
+      "name": "Rebus",
+      "display": "REBUS",
```

Comment:
the display denom should be all lowercase, as in the denom untis list; i.e., like: `"rebus"`

Reply:
addressed